gitbook/higher-order-components/mappify.md

```diff
@@ -8,6 +8,8 @@
 * The `mappify` HOC grabs a map object from the context (see also [here](https://reactjs.org/docs/context.html)) and passes it as a prop to the wrapped component
 * Commonly used in combination with the `MapProvider`
 
+**Task:** Update you app by mappifying your components.
```

Comment:
```suggestion
**Task:** Update your app by mappifying your components.
```

Reply:
fixed in https://github.com/terrestris/react-geo-ws/pull/15/commits/5f38d242d360fc0af136f4d799f4783275692b80